tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -806,6 +814,81 @@ function ( $asset ) {
 		);
 	}
 
+	public function test_non_sk_user_scripts_not_enqueued_for_sk_users() {
+		$this->enable_feature( 'rrmModuleV2' );
+
+		// Create a user.
+		$user = $this->factory()->user->create_and_get( array( 'role' => 'editor' ) );
+		$user->add_cap( Permissions::VIEW_SPLASH );
+		wp_set_current_user( $user->ID );
+
+		$dismissed_items = new \Google\Site_Kit\Core\Dismissals\Dismissed_Items( $this->user_options );
+		$dismissed_items->add( 'shared_dashboard_splash' );
+
+		$user->add_role( 'editor' );
```

Comment:
According to my tests, a lot of the above doesn't seem to be necessary.

```suggestion
		// Create a user.
		$user = $this->factory()->user->create_and_get( array( 'role' => 'editor' ) );
		wp_set_current_user( $user->ID );
```